tests/test_element.py

```diff
@@ -75,3 +75,18 @@ def test_electrophilicity(symbol):
 
     e = element(symbol)
     e.electrophilicity()
+
```

Comment:
Please add a test that uses elements as dict keys, to test the hash method.

Reply:
@kalvdans there are two tests added, one testing `__eq__` method and the other testing the `__ne__` method. Could you elaborate on 

> uses elements as dict keys, to test the hash method.

Reply:
I had in mind just a simple test just to make sure no one removes the __hash__ function in the future:
```
def test_hashable():
    d = {Element(18), Element(19)}
    assert Element(18) in d
    assert Element(20) not in d
```

The test for equality will not break if someone removes hash ability.

Reply:
Good point. I think we might be safe since `test__eq__` should fail without the `__hash__` function due to the nested attributes being unhashable.